File: src/main/java/sc02/Q02.java
package sc02;

import java.io.BufferedReader;
import java.io.InputStreamReader;

// A ~ : 65 ~ 90
// a ~ : 97 ~ 122

public class Q02 {
    public static void main(String[] args) {
        try (BufferedReader br = new BufferedReader(new InputStreamReader(System.in))) {
            String input = br.readLine();

            StringBuilder result = new StringBuilder();
            for (char ch : input.toCharArray()) {
                if (ch >= 65 && ch <= 90) {
                    result.append((char) (ch + 32));
                } else {
                    result.append((char) (ch - 32));
                }
            }
            System.out.println(result);
        } catch (Exception e) {
            throw new RuntimeException(e);
        }
    }
}

File: src/main/kotlin/sc02/Q02.kt
package jp.co.kopher.sc02

import java.io.BufferedReader
import java.io.IOException
import java.io.InputStreamReader

// A ~ : 65 ~ 90
// a ~ : 97 ~ 122

class Q02

fun main() {
    try {
        BufferedReader(InputStreamReader(System.`in`)).use { br ->
            val input = br.readLine()
            val result = input.chars()
                .map { ch ->
                    if (ch in 65..90) {
                        ch + 32
                    } else {
                        ch - 32
                    }
                }
                .toArray()
            val output = String(result, 0, result.size)
            println(output)
        }
    } catch (e: IOException) {
        throw RuntimeException(e)
    }
}